gp-templates/translations.php

```diff
@@ -32,6 +32,7 @@
 		<option value="" selected="selected"><?php _e( 'Bulk Actions', 'glotpress' ); ?></option>
 		<option value="approve"><?php _e( 'Approve', 'glotpress' ); ?></option>
 		<option value="reject"><?php _e( 'Reject', 'glotpress' ); ?></option>
+		<option value="fuzzy"><?php _e( 'Fuzzy', 'glotpress' ); ?></option>
```

Comment:
My reason was that Approve (status is `current`) and Reject sound more like an action, Fuzzy is a status.